src/web/client/test/integration/commonUtil.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { expect } from "chai";
+import {
+    convertfromBase64ToString,
+    convertStringtoBase64,
+    GetFileNameWithExtension,
+} from "../../utilities/commonUtil";
+
+describe("commonUtil", async () => {
+    it("it should return text from base 64", () => {
+        const str = "this is test case";
```

Comment:
None of the comments are incorporated in this PR. Could you follow the best practices in this PR too which I have shared over mail?

Reply:
Thanks Bidisha. This PR raised before the comments will implement the comments here as well .